ids/node_id.go

```diff
@@ -4,68 +4,62 @@
 package ids
 
 import (
-	"bytes"
```

Comment:
this may appear confusing but the content of previous version of node_id.go file is in short_node_id.go. I have renamaned what what ids.NodeID to ids.ShortNodeID and introduce a new node id type which I named ids.NodeID